averages.js

```diff
@@ -0,0 +1,46 @@
+function total(array) {
+  var sum = 0
+  for (var i=0; i < array.length; i++) {
+    sum += array[i]
```

Comment:
Thinking forward - is there an Array.prototype  method that could replace this for loop?